include/bpf/libbpf.h

```diff
@@ -915,6 +915,28 @@ void
 ring_buffer__free(struct ring_buffer* rb);
 /** @} */
 
+/**
+ * @brief Query the BPF program flags.
```

Comment:
What numbering space (or macro prefix) do these flags use?

Reply:
These flags aren't defined by eBPF core. They are a shared value between the libbpf consumer and the extension developer. The same flag value can have different meanings depending on the program type.

Reply:
Add to comment explaining what can be used here (or a way to look up what can be used here).

Reply:
Added explanation to the set API.